app/assets/stylesheets/posting.css.scss

```diff
@@ -1,3 +1,40 @@
-// Place all the styles related to the Posting controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+.posting {
+	height: 100px;
+}
+
+.postingLeft {
+	position: relative;
+	display: inline-block;
+	width: 300px;
+	height: 100px;
+}
+
+.title {
+	height: 30px;	
+}
+
+.description {
+	height: 45px;	
+}
+
+.tagBox {
+	text-align: center;
+	width: 75px;
+	background-color: #ddd;
+}
+
+.location {
+	display: inline-block;
+	width: 150px;
+	height: 100px;
+	text-align: center;
+	vertical-align: middle;
+}
+
+.date {
```

Comment:
one thing i'd change is to make these css classes more specific to make sure there are no conflicts elsewhere in the app.  

i.e. 
`.date` -> `.posting-date`
`.location` -> `.posting-location`
etc